imagersite/imagersite/static/styles/_base.scss

```diff
@@ -0,0 +1,40 @@
+
+p, a {
+  font-family: 'Montserrat';
```

Comment:
Don't forget to use your `$main-font` variable instead of hard-coding things like this.